velox/functions/prestosql/json/SIMDJsonUtil.h

```diff
@@ -47,4 +47,9 @@ void simdjsonErrorsToExceptions(
 simdjson::simdjson_result<simdjson::ondemand::document> simdjsonParse(
     const simdjson::padded_string_view& json);
 
+/// Parse the input json string using a thread local on demand parser. Allow
+/// incomplete json input.
+simdjson::simdjson_result<simdjson::ondemand::document> simdjsonParseIncomplete(
+    const simdjson::padded_string_view& json);
```

Comment:
Can we add this function in Spark folder as it is only used by Spark function?

Reply:
@rui-mo, this experimental feature in simdjson was proposed by someone to fix semantic inconsistency for presto sql. The added code may be re-usable by presto in the future. So I think it's ok to keep it here.

Reply:
cc: @Yuhta